packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.3.2
+
+* Add support for Android's `WebViewSettings.setTextZoom`
```

Comment:
Since this is the Android implementation, it doesn't need to reference the platform. This should also use present tense as specified at https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style. And it also good to provide how to use the feature added.

```suggestion
* Adds support to set text zoom of a page. See `AndroidWebViewController.setTextZoom`.
```